config.md

```diff
@@ -25,8 +25,9 @@ NOTE that there is no guarantee for forward or backward compatibility for versio
 
 Each container has exactly one *root filesystem*, specified in the *root* object:
 
-* **`path`** (string, required) Specifies the path to the root filesystem for the container. A directory MUST exist at the path declared by the field.
-* **`readonly`** (bool, optional) If true then the root filesystem MUST be read-only inside the container. Defaults to false.
+* **`path`** (string, required) Specifies the path to the root filesystem for the container.
+  A directory MUST exist at the path declared by the field.
+* **`readonly`** (bool, optional) If true then the root filesystem MUST be read-only inside the container, defaults to false.
```

Comment:
I liked “Defaults to false” better as its own sentence, but I expect folks will get the idea either way ;).